contracts/Migrator.sol

```diff
@@ -127,20 +127,21 @@ contract Migrator is Governable {
         uint256 newStakeAmount,
         uint256 newStakeDuration
     ) external isSolvent {
-        if (lockupIds.length == 0) {
-            revert LockupIdsRequired();
```

Comment:
We can remove the `LockupIdsRequired` error message.

Reply:
I thought I removed it, not sure how it slipped, have removed it now